src/cpp/session/resources/terminal/hooks

```diff
@@ -3,8 +3,39 @@
 # if we have a conda prefix, ensure it's still used
 if [ -n "${_RS_CONDA_PREFIX}" ]; then
 
-	# activate the environment
-	conda activate "${_RS_CONDA_PREFIX}"
+	# ideally, we'd call conda activate here,
+	# but users (especially on Windows) might not
+	# have called 'conda init' with their shell,
+	# so we fake the process here
+	CONDA_PREFIX="${_RS_CONDA_PREFIX}"
+	export CONDA_PREFIX
+
+	# on Windows, we need to put a number of locations
+	# on the PATH for conda utilities to function
+	#
+	# in particular, conda's crypto libraries need to
+	# be on the PATH for 'conda install' to function
+	case "$(uname -s)" in
+	MINGW32*)
+		
+		_CONDA_PREFIX=$(cygpath "${CONDA_PREFIX}")
```

Comment:
We might need something similar on WSL; note that `cygpath` isn't available by default there.